src/app/services/feathers.service.ts

```diff
@@ -1,34 +1,33 @@
 import { Injectable } from '@angular/core';
 
-import * as feathers from 'feathers/client';
 import * as feathersRx from 'feathers-reactive';
 import * as io from 'socket.io-client';
-import * as hooks from 'feathers-hooks';
-import * as socketio from 'feathers-socketio/client';
-import * as authentication from 'feathers-authentication-client';
 
+import feathers from '@feathersjs/feathers';
```

Comment:
Does the ng CLI transpiler include `node_modules`? If it works it probably does (or just works on newer browsers).

Reply:
The TypeScript default target is ES5, so `import feathers from '@feathersjs/feathers'` transpiles to `const feathers = require('@feathersjs/feathers).default`

Reply:
Sorry, I meant will the code in `@feathersjs/feathers` be transpiled as well? Some module loaders include `node_modules` by default (assuming that it is already ES5 which is not the case here).

Reply:
Feathers core is using ES6 syntax but not ES module imports.

Reply:
I checked a standard prod build and everything feathers related is still es6 (I only looked for arrow functions, though).